frontend/src/app/(public)/auth/recovery-code-error/page.tsx

```diff
@@ -45,7 +45,7 @@ export default function AuthCodeErrorPage() {
           </Typography>
 
           <Stack spacing={2} alignItems="center" sx={{ mt: 3 }}>
-            <Link href="/reset-password" style={{ textDecoration: 'none' }}>
+            <Link href="/forgot-password" style={{ textDecoration: 'none' }}>
```

Comment:
The public page path hasn't changed right? We can standardize them so both paths are `/forgot-password`

Reply:
Yes, it's "https://www.asianweddingmakeup.com/forgot-password"!

Should we change the titles to be consistent? All of them say "Reset" haha

Reply:
Oh yes let's do that!